Gopkg.toml

```diff
@@ -26,7 +26,8 @@
 
 [[constraint]]
   name = "go.mongodb.org/mongo-driver"
-  version = "^v1.1.2"
+  branch = "GODRIVER-1490"
+  source = "github.com/tfogo/mongo-go-driver"
```

Comment:
Why is this necessary? Just curious.

Reply:
I think it's because the go driver PR hasn't been merged yet